tests/bgpd/test_mp_attr.c

```diff
@@ -906,6 +925,24 @@ static struct test_segment mp_unreach_segments[] = {
 		(3 + (1 + 3 + 8 + 2) + (1 + 3 + 8 + 3)),
 		SHOULD_PARSE,
 	},
+	{
+		"IPv4",
+		"IPV4 MP Unreach, flowspec, 1 NLRI",
+		{
+			/* AFI / SAFI */ 0x0,
+			AFI_IP,
+			IANA_SAFI_FLOWSPEC,
+			0x06,
+			0x01,
+			0x1e,
+			0x1e,
+			0x28,
+			0x28,
+			0x0
+		},
```

Comment:
Same let's have some modern c here

Reply:
hard to insert modern C into a char table, without adding a new definition type.
I will keep the comments here.